gdx/src/com/badlogic/gdx/graphics/Color.java

```diff
@@ -113,6 +114,19 @@ public Color set (Color color) {
 		return this;
 	}
 
+	/** Sets this color using the red, green and blue components of the provided Color and a deviating alpha value.
+	 * 
+	 * @param rgb the desired red, green and blue values (alpha of that Color is ignored)
+	 * @param float the desired alpha value
+	 * @return this color. */
+	public Color set (Color rgb, float alpha) {
+	  this.r = rgb.r;
+	  this.g = rgb.g;
+	  this.b = rgb.b;
+	  this.a = alpha;
```

Comment:
i think that alpha should be clamped to be consistent with other setters.

Reply:
It could also call the existing set(float, float, float, float) method, which should clamp already.

Reply:
i don't think so, there is no need to clamp rgb because it's supposd to be clamped and to be consistent with setColor(Color) which doesn't clamp components.

Reply:
Good catch. I've made use of `MathUtils.clamp`. Didn't see the immediate benefit to inline it given the powerhouse that is modern JVM, though I can be convinced otherwise.

Reply:
Not sure if the dependency is desired though, that is up to you guys.

Reply:
In all of my recent code, I use `Math.min(Math.max(value, lowerBound), upperBound)` because I have noticed a measurable performance improvement on recent JVMs (16 and up, maybe slightly newer). That said, inlining is tricky, and if MathUtils.clamp() gets inlined but the core Math methods somehow do not, or if you're running on Android, iOS, or GWT, things are quite different. The clamp() approach could certainly be faster on Android, but I have no idea how to benchmark on that OS with the wide range of hardware and OS versions.